File: backend/app/models/schedule.py
from sqlalchemy import Column, Integer, String, DateTime
from app.database import Base
from datetime import datetime


class Schedule(Base):
    """일정 테이블"""
    __tablename__ = "schedules"

    id = Column(Integer, primary_key=True, index=True)
    title = Column(String, nullable=False)
    description = Column(String, default="")
    date = Column(DateTime, nullable=False)          # 일정 시작 시각
    created_at = Column(DateTime, default=datetime.utcnow)

File: app/src/main/java/com/example/capstoneassistant_na/data/model/schedule.kt
package com.example.capstoneassistant_na.data.model

data class Schedule(
    val id: Int,
    val title: String,
    val description: String,
    val date: String,       // "2025-05-23T18:00:00"
    val created_at: String
)

data class ScheduleCreate(
    val title: String,
    val description: String,
    val date: String
)